coralproject/talk
# Refactor TagsInput and improve UX

## What does this PR do?
- Upgrade `react-tags-input` to latest.
- Use `addOnBlur` per default.
- Use `react-input-autosize` to autosize the input field.
- Refactor `react-tags-input` into a `TagsInput` component.

## How do I test this PR?

- Test all 4 instances of `TagsInput` in `installer`, and `settings`
- See that tags are automatically added when blurring the focus (e.g. click outside)
- See that when I type in a long text, the input autoresizes until a full line.
